src/System.Threading.Overlapped/tests/ThreadPoolBoundHandle_Helpers.cs

```diff
@@ -19,14 +20,14 @@ struct NonBlittableType
         public string s;
     }
 
-    private static ThreadPoolBoundHandle CreateThreadPoolBoundHandle()
+    private ThreadPoolBoundHandle CreateThreadPoolBoundHandle()
     {
         return CreateThreadPoolBoundHandle((SafeHandle)null);
     }
 
-    private static ThreadPoolBoundHandle CreateThreadPoolBoundHandle(SafeHandle handle)
+    private ThreadPoolBoundHandle CreateThreadPoolBoundHandle(SafeHandle handle)
     {
-        handle = handle ?? HandleFactory.CreateAsyncFileHandleForWrite();
+        handle = handle ?? HandleFactory.CreateAsyncFileHandleForWrite(GetTestFilePath());
```

Comment:
FYI I did it here: https://github.com/dotnet/corefx/pull/21029/files#diff-2506d612f51b50d938434c157cbe72aaR21